packages/server/api/routes/users.router.js

```diff
@@ -13,4 +13,22 @@ router.post('/', async (req, res, next) => {
   }
 });
 
+router.get('/:id', (req, res) => {
+  const uid = req.params.id;
+
+  usersController
+    .getUserInformation(uid)
+    .then((result) => {
+      if (result.length === 0) {
+        res.status(404).send('Invalid user  id');
```

Comment:
picking on details here, but with 404 it should be more "couldn't find user with this id".

Ivalid means some syntax/parsing error of the string itself.